src/main/java/core/basesyntax/GetArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface GetArea {
+    int areaCalculator();
+}
```

Comment:
```suggestion
public interface AreaCalculator {
    double calculateArea();
}
```

remember that method names should be verbs, and class/interface name should be nouns